clang/lib/Serialization/ASTReader.cpp

```diff
@@ -11357,8 +11358,13 @@ void OMPClauseReader::VisitOMPAllocateClause(OMPAllocateClause *C) {
 
 void OMPClauseReader::VisitOMPNumTeamsClause(OMPNumTeamsClause *C) {
   VisitOMPClauseWithPreInit(C);
-  C->setNumTeams(Record.readSubExpr());
   C->setLParenLoc(Record.readSourceLocation());
+  unsigned NumVars = C->varlist_size();
+  SmallVector<Expr *, 16> Vars;
+  Vars.reserve(NumVars);
+  for ([[maybe_unused]] unsigned I : llvm::seq<unsigned>(NumVars))
```

Comment:
Does this work:
```
 for (auto _ : llvm::seq<unsigned>(NumVars))
```

Reply:
I thought this is Python syntax for unused variable?

Reply:
Looks to me like it works fine in C++:
https://cxx.godbolt.org/z/7W5zxs1ME